nifi-extension-bundles/nifi-extension-utils/nifi-event-transport/src/main/java/org/apache/nifi/event/transport/netty/EventLoopGroupFactory.java

```diff
@@ -54,7 +63,11 @@ public void setWorkerThreads(final int workerThreads) {
     }
 
     protected EventLoopGroup getEventLoopGroup() {
```

Comment:
perhaps this method should be renamed from 'getEventLoopGroup' to 'createEventLoopGroup'

Reply:
That could be worth considering, given the behavior, although I think it is fine to leave the current name for now.

Reply:
I changed the method name to 'createEventLoopGroup'